merkledag/cidversion.go

```diff
@@ -0,0 +1,26 @@
+package merkledag
+
+import (
+	"fmt"
+)
+
+// CidVersion is a a cid version number that is guaranteed to be valid
+type CidVersion struct {
```

Comment:
Ahh, missed that. Ok, but I would still be for having variables aliasing usable values.